TEST_RUN_EVENTS.csv

```diff
@@ -0,0 +1,39 @@
+Test Run Events,,,
+,,,
+EVENT_SUFFIX,Description,,
+_INFO,Adding extra context for an specific task,,
+_SUCCESS,The task has finalized with no errors,,
+_ERROR,An execution problem was found,,
+_START,A new task section has begun,,
+,,,
+,,,
+,,,
+,,,
+,,,
+Stage  ,Event Type,Description,Definition
+Trigger,CREATED_INFO,Trigger Run has been created,
+Trigger,RESOLVE_ERROR,Resolving trigger details failed,
+Trigger,RESOLVE_SUCCESS,Resolving trigger details failed,
+Trigger,RESOLVE_START,Resolving trigger details based on environment variables,
+Trigger,EXECUTION_START,Initial trigger execution,
+Trigger,EXECUTION_SUCCESS,Successful trigger execution,
+Trigger,HTTP_UNREACHABLE_HOST_ERROR,Tracetest could not reach the defined host in the trigger,
+Trigger,DOCKER_COMPOSE_HOST_MISMATCH_ERROR,"We identified Tracetest is running in docker compose, to connect to the host machine use the `host.docker.internal` hostname. For more information see https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds",
+Trigger,GRPC_UNREACHABLE_HOST_ERROR,Tracetest could not reach the defined host in the trigger,
+Trace,FETCHING_START,Starting the trace fetching process,
+Trace,QUEUED_INFO,Trace Run has been queued to start the fetching process,
+Trace,DATA_STORE_CONNECTION_INFO,A Data store connection request has been executed,test connection result information
+Trace,POLLING_START,Starting the trace polling process,
+Trace,POLLING_ITERATION_INFO,A polling iteration has been executed,# of spans - iteration # - reason of next iteration
+Trace,POLLING_SUCCESS,The polling strategy has succeeded to fetch the trace from the Data Store,
+Trace,POLLING_ERROR,The polling strategy has failed to fetch the trace,
+Trace,FETCHING_SUCCESS,The trace was successfully processed by the backend,
+Trace,FETCHING_ERROR,The trace was not able to be fetched,
+Trace,STOPPED_INFO,The test run was stopped during its execution,
+Test,OUTPUT_GENERATION_WARNING,The value for output <output_name> could not be generated,

```

Comment:
If output is not generated, should this be a warning or an error? Today we keep executing the test because we don't have any way of showing the user why the test failed, but with this event system, it would be possible to halt the test and say "output X could not be generated: <more details why>" and mark the test as failed.

Reply:
That's a great point, I was thinking about the same. I'll add a ticket to the epic for Test error handling improvements as we are only covering trigger and trace, I would say that the run shouldn't be stopped tho, just having some warning or error signs around the outputs that failed. what do you think @kdhamric?

Reply:
This and everything regarding the Test section will be considered as follow-ups as we need to focus on trigger and trace for the moment.

https://github.com/kubeshop/tracetest/issues/2256
https://github.com/kubeshop/tracetest/issues/2257